contrib/opencensus-ext-azure/README.rst

```diff
@@ -114,11 +114,70 @@ You can enrich the logs with trace IDs and span IDs by using the `logging integr
         logger.warning('In the span')
     logger.warning('After the span')
 
+Metric
+~~~~~~
+
+The **OpenCensus Azure Monitor Metrics Exporter** allows you to export metrics to `Azure Monitor`_.
+
+Metrics Exporter Usage
+~~~~~~~~~~~~~~~~~~~~~~
+
+Metrics Exporter Prerequisites
+******************************
+
+* Create an Azure Monitor resource and get the instrumentation key, more information can be found `here <https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource>`_.
+* Put the instrumentation key in ``APPINSIGHTS_INSTRUMENTATIONKEY`` environment variable.
+
+Using the Metrics exporter
+*****************************
+
+    .. code:: python
+
+        import time
+
+        from opencensus.ext.azure import metric_exporter
+        from opencensus.stats import aggregation as aggregation_module
+        from opencensus.stats import measure as measure_module
+        from opencensus.stats import stats as stats_module
+        from opencensus.stats import view as view_module
+        from opencensus.tags import tag_map as tag_map_module
+
+
+        # The stats recorder
+        stats = stats_module.stats
+        view_manager = stats.view_manager
+        stats_recorder = stats.stats_recorder
+
+        PROBLEMS_SOLVED_MEASURE = measure_module.MeasureInt("problems_solved", "number of problems solved", "problems")
+        PROBLEMS_SOLVED_VIEW = view_module.View('problems_solved_view', "number of problems solved", [], PROBLEMS_SOLVED_MEASURE, aggregation_module.CountAggregation())
```

Comment:
This line is too long when you read the markdown file on GitHub:

![image](https://user-images.githubusercontent.com/17327289/59380443-fa9a0580-8d0d-11e9-9fff-76503fdd4fe7.png)